src/mozilla_sec_eia/cli.py

```diff
@@ -42,6 +47,8 @@ def formatter(prog) -> argparse.HelpFormatter:
 
     # Add command to fine-tune ex21 extractor
     validate_parser = subparsers.add_parser("finetune_ex21")
+    validate_parser.add_argument("--labeled-json-path")
+    validate_parser.add_argument("--gcs-training-data-dir", default="labeled/")
```

Comment:
Added these arguments because now there are folders with v0.0 and v0.1 of training data, and I want to be able to specify which folder to pull from.